protelis-interpreter/src/main/java/org/protelis/lang/interpreter/impl/AbstractPersistedTree.java

```diff
@@ -23,8 +23,7 @@
  */
 public abstract class AbstractPersistedTree<S, T> extends AbstractProtelisAST<T> {
 
-    private static final long serialVersionUID = 457607604000217166L;
-    private transient S superscript;
+    private static final long serialVersionUID = 1;
```

Comment:
Why this change? Seems unrelated to the rest of the pull.

Reply:
I removed a field, potentially changing the serialized format. As such, I modified the serial version UID. This would make systems running Protelis at an older version not compatible with systems using Protelis at this version (cross-version runtime compatibility) if they use the default Java serialization, falling clearly.